[32mINFO    [0m test_sign_in_password:test_E2E.py:24 Testcase: Sign in with Password
[32mINFO    [0m test_attempt_diagnostic_test_2:test_E2E.py:698 Test case: User attempts Diagnostic Test - 2